confmap/provider/httpprovider/README.md

```diff
@@ -0,0 +1,13 @@
+What is this new component httpprovider?
+- An implementation of ConfigMapProvider for HTTP (httpprovider) allows OTEL Collector the ability to load configuration for itself by fetching and reading config files stored in HTTP servers.
```

Comment:
ConfigMapProvider is the old name please update everywhere

Reply:
Fixed!